hikari/__init__.py

```diff
@@ -22,7 +22,7 @@
 # SOFTWARE.
 """A sane Python framework for writing modern Discord bots.
 
-To get started, you will want to initialize an instance of `BotApp`
+To get started, you will want to initialize an instance of `GatewayBot`
 for writing a bot, or `RESTApp` if you only need to use the REST API.
```

Comment:
Shouldnt more info on RESTBot be added here along with an example in the examples folder + README?

Reply:
can be added in another PR